packages/pyright-internal/src/tests/fourslash/completions.params.fourslash.ts

```diff
@@ -1,10 +1,26 @@
 /// <reference path="fourslash.ts" />
 
 // @filename: test.py
+//// from typing import TypedDict, Unpack
+////
+//// class Movie(TypedDict):
+////     key1: str
+////     key2: int
+////
 //// def method(param1=None, param2='active', param3=None):
 ////     pass
 ////
 //// met/*marker1*/hod   /*marker2*/ (    /*marker3*/      param2 = 'test')
+////
+//// def method2(param1: int, **kwargs: Unpack[Movie]):
+////     pass
+////
+//// method2(p/*marker4*/, k/*marker5*/)
+////
+//// def method3(param1: int, **kwargs: Any):
```

Comment:
Need to import `Any`?

Reply:
It's not necessary, but wouldn't hurt I suppose. It's just ignoring the diagnostics produced.

Reply:
It's fine either way. I only noticed when I copied this code to play with it. Then I was confused about why some things were imported but others were not. I guess `TypedDict` and `Unpack` are required for the `TypeEvaluator` to understand what's going on?

Reply:
I think so. I was just copying other tests and forgot to include 'Any' in the import list. But the test passed without that part.